cloverage/project.clj

```diff
@@ -16,11 +17,12 @@
     :scm :git ; Because we're not in the top-level directory, so it doesn't auto-detect
     :deploy-via :clojars
   }
-  :dependencies [[org.clojure/clojure "1.4.0"]
-                 [org.clojure/tools.cli "0.2.2"]
-                 [org.clojure/tools.logging "0.2.3"]
-                 [org.clojure/data.xml "0.0.7"]
-                 [bultitude "0.2.0"]
-                 [riddley "0.1.4"]
-                 [slingshot "0.10.3"]
-                 [cheshire "5.3.1"]])
+  :dependencies [[org.clojure/clojure "1.6.0"]
```

Comment:
Turns out I did it in order to get the repl working when using cider. I'll change it back.

Out of curiosity: what's the reasoning behind avoiding the upgrade?